app/javascript/components/CardColumn/Card/card-user-avatar/card-user-avatar.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {getUserInitials} from '../../../../utils/helpers';
+
+const CardUserAvatar = ({avatar, firstName, lastName}) => {
+  const renderAvatar = (userAvatar, userName, userSurname) => {
+    if (userAvatar) {
+      return <img src={userAvatar} className="avatar" />;
+    }
+
+    return (
+      <div className="avatar avatar--text">
+        {getUserInitials(userName, userSurname)}
+      </div>
+    );
+  };
+
+  return renderAvatar(avatar, firstName, lastName);
+};
```

Comment:
In this case `renderAvatar` isn't needed at all
```js
const CardUserAvatar = ({avatar, firstName, lastName}) => {
  if (avatar) {
    return <img src={avatar} className="avatar" />;
  }

  return (
    <div className="avatar avatar--text">
      {getUserInitials(firstName, lastName)}
    </div>
  );
};
```

Reply:
done